### DISCUSSION TITLE
Gmail ?

### DISCUSSION THREAD

**Discussion by DeiAsPie** [yangshun/tech-interview-handbook#382] [Q&A]
Sir
Your handbook recommends Gmail over others. What makes Gmail special ?

**Comment by yassine-krichen** [comment]
Gmail gets recommended because… it just works. That’s honestly it. Recruiters see it, their spam filters don’t freak out, and nobody has to squint at your address wondering if it’s a sketchy server in someone’s basement. It’s boring, stable, and everyone already uses it so nobody complains.

Other providers are fine, but Gmail is the default because tech people are lazy and don’t want surprises. That’s the whole “secret.”
